src/wp-includes/media.php

```diff
@@ -1993,8 +2008,14 @@ function wp_img_tag_add_auto_sizes( string $image ): string {
 	}
 
 	// Bail early if the image is not lazy-loaded.
-	$value = $processor->get_attribute( 'loading' );
-	if ( ! is_string( $value ) || 'lazy' !== strtolower( trim( $value, " \t\f\r\n" ) ) ) {
+	$loading = $processor->get_attribute( 'loading' );
+	if ( ! is_string( $loading ) || 'lazy' !== strtolower( trim( $loading, " \t\f\r\n" ) ) ) {
+		return $image;
+	}
+
+	// Bail early if the image doesn't have a width attribute.
+	$width = $processor->get_attribute( 'width' );
+	if ( ! is_string( $width ) ) {
```

Comment:
Is it possible for `$width` to be an empty string here? Perhaps `empty( $width ) || ! is_string( $width )` may be better.

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/7813/commits/baee465a64720148002dd71296f914df74dd1df0